src/main/java/core/basesyntax/service/impl/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriterService;
+import core.basesyntax.storage.Storage;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class FileWriterServiceImpl implements FileWriterService {
+    private static final String FIRST_LINE = "fruit,quantity" + System.lineSeparator();
+    private static final String COMA_SEPARATOR = ",";
+
+    @Override
+    public void writeToFile(String path) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(path))) {
+            bufferedWriter.write(FIRST_LINE);
+            for (Map.Entry<String, Integer> entry : Storage.dataStorage.entrySet()) {
+                StringBuilder builder = new StringBuilder();
+                builder.append(entry.getKey())
+                        .append(COMA_SEPARATOR)
+                        .append(entry.getValue())
+                        .append(System.lineSeparator());
+                bufferedWriter.write(builder.toString());
+            }
```

Comment:
Let's create a result string (the data that should be written to the file) using a separate service (for example, ReportService) and pass it to this method